addons/account_edi/models/account_edi_format.py

```diff
@@ -89,6 +89,11 @@ def _needs_web_services(self):
         self.ensure_one()
         return False
 
+    def _allowed_force_cancel(self):
```

Comment:
Instead of having this `_allowed_force_cancel` hook on `account.edi.format`, I would instead create a hook on `account.move` (e.g. called `_can_force_cancel`) that returns False, and would be inherited by your other modules.

That would provide improved flexibility for filtering invoices that are allowed to be force-cancelled according to any criteria (e.g. how long has passed since the cancel request, etc.).